staging/src/k8s.io/component-base/metrics/counter_test.go

```diff
@@ -286,3 +289,94 @@ func TestCounterWithLabelValueAllowList(t *testing.T) {
 		})
 	}
 }
+
+func TestCounterWithExemplar(t *testing.T) {
+	// Set exemplar.
+	fn := func(offset int) []byte {
+		arr := make([]byte, 16)
+		for i := 0; i < 16; i++ {
+			arr[i] = byte(2<<7 - i - offset)
+		}
+		return arr
+	}
+	traceID := trace.TraceID(fn(1))
+	spanID := trace.SpanID(fn(2))
+	ctxForSpanCtx := trace.ContextWithSpanContext(context.Background(), trace.NewSpanContext(trace.SpanContextConfig{
+		TraceID: traceID,
+		SpanID:  spanID,
```

Comment:
```suggestion
		SpanID:  spanID,
		TraceFlags: trace.FlagsSampled,
```
If you accept the other suggestion, you will need to update the test.